### DISCUSSION TITLE
Selfhosted migrations without postgres superuser

### DISCUSSION THREAD

**Discussion by tonyjara** [supabase/supabase#10372] [Questions] [answered]
Hi, I'm currently trying to self-host supabase with a managed DB. Sadly my organization is tied to providers that don't offer superuser capabilities on their managed postgres instances. I just want to know if it is posible to run the migrations necessary for supabase to work and supabase in general without superuser. If it's going to be a pain further down I'll just manage the db myself.

**Comment by awalias** [comment, marked as answer]
it's possible to get some of the functionality but you're likely going to run into issues with any feature that requires postgres extensions not supported by your managed provider (or that they're unwilling to install for you)

**Reply by tonyjara** [reply]
Thank you, that answers my question, I will be moving away from this approach.
